stock_warehouse_flow/views/stock_warehouse_flow.xml

```diff
@@ -16,30 +16,35 @@
                         attrs="{'invisible': [('active', '=', True)]}"
                     />
           <div class="oe_title">
-              <label for="name" class="oe_edit_only" />
-              <h1><field name="name" placeholder="Flow Name" /></h1>
+            <label for="name" />
+            <h1>
+              <field name="name" placeholder="Flow Name" />
+            </h1>
           </div>
           <group>
-            <group name="info" colspan="4" col="4">
-              <field
-                                name="warehouse_id"
-                                attrs="{'readonly': [('delivery_route_id', '!=', False)]}"
-                            />
-              <field name="sequence" />
-              <field name="active" invisible="1" />
-              <field name="company_id" invisible="1" />
+            <group name="info" colspan="4">
+              <group>
+                <field
+                                    name="warehouse_id"
+                                    attrs="{'readonly': [('delivery_route_id', '!=', False)]}"
+                                />
+              </group>
+              <group>
+                <field name="sequence" />
+                <field name="active" invisible="1" />
+                <field name="company_id" invisible="1" />
+              </group>
```

Comment:
Not blocking. Are these view changes required for the migration? (Odoo changes the way it is rendered and without it it's "ugly"?).
If not this deserves to be in its own commit to be backported.